cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -268,8 +268,8 @@ function Set-EnvironmentVars {
     "KUBELET_CERT_PATH" = ${kube_env}['PKI_DIR'] + '\kubelet.crt'
     "KUBELET_KEY_PATH" = ${kube_env}['PKI_DIR'] + '\kubelet.key'
 
-    "CONTAINER_RUNTIME" = ${kube_env}['CONTAINER_RUNTIME']
-    "CONTAINER_RUNTIME_ENDPOINT" = ${kube_env}['CONTAINER_RUNTIME_ENDPOINT']
+    "WINDOWS_CONTAINER_RUNTIME" = ${kube_env}['WINDOWS_CONTAINER_RUNTIME']
+    "WINDOWS_CONTAINER_RUNTIME_ENDPOINT" = ${kube_env}['WINDOWS_CONTAINER_RUNTIME_ENDPOINT']
```

Comment:
To be extra defensive, could you update the code for each of these two variables to be:

```
if ($kube_env.ContainsKey('WINDOWS_CONTAINER_RUNTIME')) {
    "CONTAINER_RUNTIME" = ${kube_env}['WINDOWS_CONTAINER_RUNTIME']
} else {
    Log-Output "ERROR: WINDOWS_CONTAINER_RUNTIME not set in kube-env, falling back in CONTAINER_RUNTIME"
    "CONTAINER_RUNTIME" = ${kube_env}['CONTAINER_RUNTIME']
}
```

This will avoid fatal errors if we mess up the versioning between the node and the control plane (e.g. when cherry-picking this change).

To make this work you may need to introduce local variables above, before the $env_vars hash table definition.